tests/Domain/NoteTest.php

```diff
@@ -0,0 +1,307 @@
+<?php
+namespace Notes\Domain;
+
+use Notes\Model\Note as NoteModel;
+use Notes\Domain\Note as NoteDomain;
+use Notes\Config\Config as Configuration;
+use Notes\Model\UserTag as UserTagModel;
+use Notes\Model\NoteTag as NoteTagModel;
+use Notes\Model\User as UserModel;
+use Notes\Collection\Collection as Collection;
+
+class NoteTest extends \PHPUnit_Extensions_Database_TestCase
+{
+    private $connection;
+    
+    public function getConnection()
+    {
+        $config     = new Configuration("app/Config/config.json");
+        $configData = $config->get();
+        $dbHost     = $configData['dbHost'];
+        $dbName     = $configData['dbName'];
+        $hostString = "mysql:host=$dbHost;dbname=$dbName";
+        
+        try {
+            $this->connection = new \PDO($hostString, $configData['dbUser'], $configData['dbPassword']);
+            $this->connection->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
+            $this->connection->exec("set foreign_key_checks=0");
+            return $this->createDefaultDBConnection($this->connection, $dbName);
+        } catch (\PDOException $e) {
+            echo "Connection failed: " . $e->getMessage();
+        }
+    }
+    
+    public function getDataSet()
+    {
+        return $this->createXMLDataSet(dirname(__FILE__) . '/_files/domain_note_seed.xml');
+    }
+    
+    public function testCanCreate()
```

Comment:
remove the test word add it in comments like  /**
    * @test
    *
    **/
and you can change the name of test as it_should_create_note